test/ra_log_2_SUITE.erl

```diff
@@ -426,6 +427,62 @@ written_event_after_snapshot(Config) ->
     % false = filelib:is_file(Snap1),
     ok.
 
+writes_lower_than_snapshot_index_are_dropped(Config) ->
+    Log0 = ra_log_init(Config, #{min_snapshot_interval => 1}),
+    Log1 = ra_log:append({1, 1, <<"one">>}, Log0),
+    Log1b = deliver_all_log_events(ra_log:append({2, 1, <<"two">>}, Log1), 500),
+    true = erlang:suspend_process(whereis(ra_log_wal)),
+    Log2 = write_n(3, 500, 1, Log1b),
+    {Log3, _} = ra_log:update_release_cursor(100, #{}, 1,
+                                             <<"100">>, Log2),
+    Log4 = deliver_all_log_events(Log3, 500),
+    %% TODO: assertion
```

Comment:
Is this todo resolved with the `?assertMatch` below?

Reply:
yep!